### PR TITLE
[CLOUDTRUST-2125] record an event when execute actions emails is called

### PR DESCRIPTION
**Pull request by bsoniam**

### CODE REVIEW

File: pkg/management/component.go [hunk 1]
```diff
@@ -589,15 +589,24 @@ func (c *component) ExecuteActionsEmail(ctx context.Context, realmName string, u
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 
 	var actions = []string{}
-
+	var eventRecorded bool = false
```
Comment (fperot74) [inline pkg/management/component.go]: `var eventRecorded = false` should be enough !?

File: pkg/management/component_test.go [hunk 2]
```diff
@@ -1596,6 +1596,18 @@ func TestExecuteActionsEmail(t *testing.T) {
 
 		assert.NotNil(t, err)
 	}
+	// Send email actions, but not sms-password-set
+	{
+
```
Comment (fperot74) [inline pkg/management/component_test.go]: `var actions2` and `var reqActions2` are only used here... they could be declared here

File: pkg/management/component.go [hunk 3]
```diff
@@ -589,15 +589,24 @@ func (c *component) ExecuteActionsEmail(ctx context.Context, realmName string, u
...
 	for _, requiredAction := range requiredActions {
 		actions = append(actions, string(requiredAction))
 		if string(requiredAction) == initPasswordAction {
 			//store the API call into the DB
 			c.reportEvent(ctx, "INIT_PASSWORD", database.CtEventRealmName, realmName, database.CtEventUserID, userID)
+			eventRecorded = true
 		}
 	}
 
+	if !eventRecorded {
```
Comment (harture) [inline pkg/management/component.go]: With this mechanism, the password init could hide some other required action.
What do you think about record the ACTION_EMAIL in any case ?

File: pkg/management/component.go (hunk 1 above)
Comment (bsoniam) [reply pkg/management/component.go]: Should be enough, no? Do you see a problem here ?

File: pkg/management/component_test.go (hunk 2 above)
Comment (bsoniam) [reply pkg/management/component_test.go]: Thank you for the suggestion
DONE

File: pkg/management/component.go (hunk 3 above)
Comment (bsoniam) [reply pkg/management/component.go]: Yes, we can do this
DONE

**Pull request merged**